iam-login-service/src/main/java/it/infn/mw/iam/api/scim/updater/UpdaterType.java

```diff
@@ -21,7 +21,8 @@
   ACCOUNT_REMOVE_X509_CERTIFICATE("Remove user's x509 certificate"),
 
   ACCOUNT_ADD_GROUP_MEMBERSHIP("Add group membership to user"),
-  ACCOUNT_REMOVE_GROUP_MEMBERSHIP("Remove user's group membership");
```

Comment:
Remove all the 's. 
Examples: 
- Remove user's group membership -> Remove user from group
- Remove user's X.509 certificate -> Remove X.509 certificate from user